### DISCUSSION TITLE
latent-calendar/modules/extensions/

### DISCUSSION THREAD

**Discussion by giscus** [williambdean/latent-calendar#21] [General]
# latent-calendar/modules/extensions/

https://wd60622.github.io/latent-calendar/modules/extensions/

**Comment by galenseilis** [comment]
I just came from [* Latent Calendar: Modeling Weekly Behavior with Latent Components *](https://wd60622.github.io/latent-calendar/examples/cal-attribute/). This is a very interesting project. I work in health care where we have a lot of this type of data. Assuming it passes our software security checks, I look forward to trying it out!

**Reply by williambdean** [reply]
Sounds like an interesting use case. I can imagine restrictions if the application is too personal or revealing. I have found this technique insightful to learning patterns in not only user level but location or point-in-time behavior too. 

Hope it goes well. If you have any successes you are able to share, I would love to hear
